test/support/page_objects/discover_page.js

```diff
@@ -24,7 +24,7 @@ export default class DiscoverPage {
 
   getTimespanText() {
     return this.findTimeout
-    .findByCssSelector('.kibana-nav-options .navbar-timepicker-time-desc pretty-duration')
+    .findByCssSelector('[data-test-subj="globalTimepickerRange"]')
```

Comment:
This, and the rest of the `[data-test-subj="xyz"]` selectors should be using `PageObjects.common.findTestSubject('xyz')`